### PR TITLE
Issues/724

### LINKED ISSUE #724: Setting "Success" and "Failed" pages to the same page results in Failed status with PayPal Standard
As reported here:
https://wordpress.org/support/topic/donation-transaction-failed-receipt-sent-regardless?replies=7#post-8636047

This problem exists in EDD as well:
https://easydigitaldownloads.com/forums/topic/payment-status-failed-although-all-ok-in-paypal/
# Replication Steps
- Go to "Donations > Settings", then the General tab
- Set the "Success Page" and the "Failed Transaction Page" to be the same page
- Do a PayPal Standard Donation
- Click the link to return to the website
- On the Donation Confirmation page you'll see that the status is "Failed" even though the money does get transferred correctly from the donor to the organization
# Suggestion

I think we should trigger a big red alert above the "Failed Transaction Page" setting if it gets set to the exact same page as the "Success Page"... and potentially prevent the settings from being saved. It's a serious issue since it will appear that ALL donations are failing, so preventing this from happening is important.

### PR DESCRIPTION
**Pull request by ravinderk**
This pr will fix #724.
## Description

Prevent admin from saving same success and failure page. I added backend and frontend validation to prevent this.
## How Has This Been Tested?

By saving the same page for success and failure setting.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: assets/js/admin/admin-scripts.js [hunk 1]
```diff
@@ -343,8 +340,24 @@ jQuery.noConflict();
...
+		verify_settings: function () {
+			var success_setting = $('#success_page');
+			var failure_setting = $('#failure_page');
+
+			/**
+			 * Verify success and failure page.
+			 */
+			success_setting.add(failure_setting).change(function () {
+				if (success_setting.val() === failure_setting.val()) {
+					alert(give_vars.matched_success_failure_page);
```
Comment (DevinWalker) [inline assets/js/admin/admin-scripts.js]: Rather than an alert can we display the warning on screen using a WP notice that is dismissible?

Comment (ravinderk) [inline assets/js/admin/admin-scripts.js]: @DevinWalker Fixed.

**Comment by DevinWalker** [comment]
I tested this and resolved the conflict and am merging via  https://github.com/WordImpress/Give/pull/1203

Thanks @ravinderk !

**Comment by mathetos** [comment]
Woot! Glad to see this making it in. Not a major support issue, but it has come up now and again.

**Pull request closed without merging**